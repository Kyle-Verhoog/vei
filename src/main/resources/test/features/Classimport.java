import java.util.Vector;

public class Classimport {
  public Classimport() {}
  protected Vector c;
}