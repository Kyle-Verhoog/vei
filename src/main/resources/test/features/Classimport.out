IMPORT ┃ import
IDENTIFIER ┃ java
. ┃ .
IDENTIFIER ┃ util
. ┃ .
IDENTIFIER ┃ Vector
; ┃ ;
PUBLIC ┃ public
CLASS ┃ class
IDENTIFIER ┃ A
{ ┃ {
PUBLIC ┃ public
IDENTIFIER ┃ A
( ┃ (
) ┃ )
{ ┃ {
} ┃ }
PROTECTED ┃ protected
IDENTIFIER ┃ Vector
IDENTIFIER ┃ c
; ┃ ;
} ┃ }